diff --git a/node_modules/react-jsonschema-form/lib/components/fields/MultiSchemaField.js b/node_modules/react-jsonschema-form/lib/components/fields/MultiSchemaField.js
index 4ef5868..38a4650 100644
--- a/node_modules/react-jsonschema-form/lib/components/fields/MultiSchemaField.js
+++ b/node_modules/react-jsonschema-form/lib/components/fields/MultiSchemaField.js
@@ -193,9 +193,9 @@ function (_Component) {
           value: index
         };
       });
-      return _react["default"].createElement("div", {
+      return _react["default"].createElement(_react.Fragment, {
         className: "panel panel-default panel-body"
-      }, _react["default"].createElement("div", {
+      }, _react["default"].createElement(_react.Fragment, {
         className: "form-group"
       }, _react["default"].createElement(Widget, (0, _extends2["default"])({
         id: "".concat(idSchema.$id, "_anyof_select"),
